frontend/app/model/RichEvent.scala

```diff
@@ -64,7 +64,7 @@ object RichEvent {
   }
 
   trait RichEvent {
-    val event: EBEvent
+    val underlying: EventWithDescription
```

Comment:
we ended up with `event.event.event.id` in many places, so I changed this to `underlying` and the model event to `ebEvent`.  Any alternative suggestions would be taken though.